android-sdk/README.md

```diff
@@ -0,0 +1,168 @@
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+repositories {
+   maven { url 'http://artifacts.adsmoloco.com/artifactory/libs-release-local/' }
+}
+
+dependencies {
+   compile fileTree(dir: 'libs', include: ['*.jar'])
+   implementation 'com.moloco.sdk:moloco-sdk-base:0.0.1@aar'
+   implementation 'com.moloco.sdk:moloco-sdk-banner:0.0.1@aar'
+
+   implementation 'com.squareup.retrofit2:retrofit:2.0.2'
+   implementation 'com.squareup.retrofit2:converter-gson:2.1.0'
+   implementation 'com.squareup.retrofit2:converter-scalars:2.5.0'
+   implementation 'com.squareup.retrofit2:adapter-rxjava2:2.4.0'
+   implementation 'com.google.android.gms:play-services-ads:15.0.1'
+   implementation 'io.reactivex.rxjava2:rxandroid:2.0.2'
+}
+```
+
+Now, you are ready to integrate the Moloco Android SDK into your app.
+
+
+## Integration
+### Import
+You should import classes as below.
+```
+import com.moloco.ads.MolocoView;
+import com.moloco.ads.MolocoView.BannerAdListener;
+import com.moloco.common.Moloco;
+import com.moloco.common.MolocoErrorCode;
+import com.moloco.common.SdkConfiguration;
+import com.moloco.network.DisposableManager;
+
+import static com.moloco.common.logging.MLog.LogLevel;
+import static com.moloco.common.logging.MLog.LogLevel.DEBUG;
+import static com.moloco.common.logging.MLog.LogLevel.INFO;
+```
+
+### Activity class setting.
+In an Activity (inherited) class, you should implement a BannerAdListener interface as below.
+
+```
+public interface BannerAdListener {
+    void onBannerLoaded(MolocoView banner);
+    void onBannerFailed(MolocoView banner, MolocoErrorCode errorCode);
+    void onBannerClicked(MolocoView banner);
+    void onBannerExpanded(MolocoView banner);
+    void onBannerCollapsed(MolocoView banner);
+}
+```
+
+You should add implementations in the onCreate method.
+```
+protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    /*
+        Your logic comes here
+    */
+}
+```
+
+### Initialize MolocoView
+
+```
+MolocoView mMolocoView = findViewById( /* Id of your view. ex) R.id.banner_molocoview */ );
+mMolocoView.setWidthPixel(720);
+mMolocoView.setHeightPixel(160);
+```
+
+### SdkConfiguration
+Initialize a SdkConfiguration with a LogLevel (INFO or DEBUG).
+
+```
+LogLevel logLevel = INFO; // or DEBUG
+final SdkConfiguration sdkConfiguration = new SdkConfiguration(logLevel);
+Moloco.initializeSdk(this, sdkConfiguration);
+```
+
+###  Load an Ad with an ad unit id.
+Initialization requires a `AdUnit ID` provided by Moloco. Please contact [Moloco](mailto:support@molocoads.com) if you need an `AdUnit ID`.
+
+```
+String adUnitId = "your_ad_unit_id";
+mMolocoView.setAdUnitId(adUnitId);
+mMolocoView.loadAd();
+mMolocoView.setBannerAdListener(this);
+```
+
+Now you are ready use **Moloco Android SDK** for Android devices!
+
+Once you complete integration, please work with your Moloco contact to verify receipt of the session and other events from Android server.
```

Comment:
This needs to be rewritten

Reply:
Are there any problems if I remove this line on the first version of document?

Reply:
Sounds good to me

Reply:
Android server? Does it have to be Moloco Server? Ah, it will be removed.